site/about/index.html

```diff
@@ -1,14 +1,153 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <title>Your Website Name</title>
-    <link rel="stylesheet" href="/assets/css/reset.css" type="text/css" />
-    <link rel="stylesheet" href="/assets/css/global.css" type="text/css" />
-    <link rel="stylesheet" href="styles.css" type="text/css" />
-  </head>
-  <body>
-    <nav><a href="/">Home</a><a href="/about">About</a></nav>
-    <h1>About</h1>
-  </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>about</title>
+  <link rel="stylesheet" href="/assets/css/reset.css" type="text/css" />
+  <link rel="stylesheet" href="/assets/css/global.css" type="text/css" />
+  <link rel="stylesheet" href="styles.css" type="text/css" />
+</head>
+
+<body>
+  <nav><a href="/">Home</a><a href="/about">About</a></nav>
+  <h1>Main</h1>
+  <div class="wrapper">
+    <div class="nav-1">
+      <div class="language">
```

Comment:
I think we spoke last week about this. Instead of giving each individual item it's own class, try and give all things that are alike the same class name. So these could be `class="nav-item"` for example